linkerd/app/integration/src/client.rs

```diff
@@ -119,7 +119,6 @@ impl Client {
         }
     }
 
-    #[track_caller]
```

Comment:
why was this removed? it should result in nicer panic messages if a request gets a non-200 status...

Reply:
It does nothing with async fns